### PR TITLE
Add a tsconfig for developer convenience

### PR DESCRIPTION
**Pull request by slomo**
- enables compilation via bin/tsc (without arguments)

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "compilerOptions": {
+    "module": "umd",
+    "removeComments": true,
+    "sourceMap": true
+  },
+  "files": [
+    "src/current/current/build/js/Adhocracy.ts",
+    "src/current/current/build/js/AdhocracySDK.ts",
+    "src/current/current/build/js/AdhocracySpec.ts",
+    "src/current/current/build/js/AdhocracySDKSpec.ts"
```
Comment (xi) [inline tsconfig.json]: This line is not required, because it is imported in AdhocracySpec.ts

**Comment by xi** [comment]
Should we update any documentation?

**Comment by der-john** [comment]
As far as I can see, there is no documentation of ours about this.

**Comment by slomo** [comment]
I added something to the development config.

**Comment by xi** [comment]
Sorry for nitpicking. I remember that `--watch` had some issues with symlinks in the past. We should check whether these still exist. I can do that later.

**Comment by slomo** [comment]
I am using watch. It works even if I change stuff in core, only downside that it is horribly slow, but there are already tickets on the typescript repo for this.

**Comment by xi** [comment]
Thanks for checking

**Pull request merged**